Formula/openvino.rb

```diff
@@ -0,0 +1,171 @@
+class Openvino < Formula
+  desc "Open Visual Inference And Optimization toolkit for AI inference"
+  homepage "https://docs.openvino.ai"
+  url "https://github.com/openvinotoolkit/openvino/archive/refs/tags/2022.3.0.tar.gz"
+  sha256 "b8dc1880d9ab71bd24aa5b2565724c12fb78172613e23ce312e0b98a6d8a0dd7"
+  license "Apache-2.0"
+  head "https://github.com/openvinotoolkit/openvino.git", branch: "master", using: :git
+
+  livecheck do
+    url :stable
+    regex(/^(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "pkg-config" => [:build, :test]
```

Comment:
Since the test doesn't call them, this looks like a runtime dependency

Reply:
How about CMake though?

Reply:
cmake interface is also present, but for the sake of code in tests I used gcc command directly

Reply:
thanks for the valuable comments!
the second test also uses cmake interface, while the first one was rewritten to fully rely on pkg-config files.